vm-virtio/src/vhost_user/blk.rs

```diff
@@ -214,8 +219,18 @@ impl VirtioDevice for Blk {
             error!("Failed to write config space");
             return;
         }
-        let (_, right) = self.config_space.split_at_mut(offset as usize);
-        right.copy_from_slice(&data[..]);
+        // In fact, write_config() only handle wce value in vhost-user-blk.
+        // so, we can only set wce value here.
+        // Since vhost_set_config callback function in SPDK is NULL, which show
+        // there is not any handle in SPDK side for set_config operation. The
+        // detailed info ref spdk_extern_vhost_pre_msg_handler() in SPDK.
```

Comment:
Could you please remove specific comments about SPDK?
Also, you're mentioning `wce` should be the only value being set from the guest, but with this current implementation, you're letting any config entry being updated from the guest.

Last thing, this commit should be squashed with the one introducing wce.

Reply:
yes, remove SPDK related log, thanks